Sources/Rendering/Core/TextProperty/index.js

```diff
@@ -0,0 +1,64 @@
+import macro from 'vtk.js/Sources/macros';
+import vtkProperty2D from 'vtk.js/Sources/Rendering/Core/Property2D';
+
+// ----------------------------------------------------------------------------
+// vtkTextProperty methods
+// ----------------------------------------------------------------------------
+
+function vtkTextProperty(publicAPI, model) {
+  // Set our className
+  model.classHierarchy.push('vtkTextProperty');
+}
+
+// ----------------------------------------------------------------------------
+// Object factory
+// ----------------------------------------------------------------------------
+
+const DEFAULT_VALUES = {
+  fontFamily: 'Arial',
+  fontColor: [0, 0, 0],
+  fontStyle: 'normal',
+  backgroundColor: [1, 1, 1],
```

Comment:
by default, the background color should be transparent.
Maybe no `backgroundColor` (undefined or null) could also mean translucent